### DISCUSSION TITLE
🚀 YARP 3.2.2 Released

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#370] [Releases]
YARP 3.2.2 was released today and is now available for download at: https://github.com/robotology/yarp/releases/.

A (partial) list of bug fixed and issues resolved in this release can be found
[here](https://github.com/robotology/yarp/issues?q=label%3A%22Fixed+in%3A+YARP+v3.2.2%22).

Bug Fixes
---------

### Build System

* OpenCV 4 is not supported yet, and rejected by the build system.

#### `ovrheadset`

* Added the hands_relative parameter (robotology/yarp#2106)

#### `USBCameraDriverRaw`

* Do not try to build when missing dependencies

#### `yarpdataplayer`

* Fixed speed control not working

Contributors
------------

This is a list of people that contributed to this release (generated from the
git history using `git shortlog -ens --no-merges v3.2.1..v3.2.2`):

```
     7	Daniele E. Domenichelli <daniele.domenichelli@iit.it>
     3	Vadim Tikhanoff <vadim.tikhanoff@iit.it>
     1	Giulio Romualdi <giulio.romualdi@gmail.com>
     1	Marco Randazzo <marco.randazzo@iit.it>
```

**Comment by lornat75** [comment]
thanks all!
